tests/spec/test_spec.py

```diff
@@ -16,13 +18,19 @@
 
 
 def pytest_generate_tests(metafunc):
+    skip_slow_spec = metafunc.config.getoption('skip_slow_spec')
     generate_fixture_tests(
         metafunc=metafunc,
         fixtures_dir=FIXTURES_DIR,
+        skip_slow=skip_slow_spec,
     )
 
 
-def test_json_fixture(fixture_path):
+def test_json_fixture(fixture_path, pytestconfig):
+    # ensure that all numpy flags are set to raise exceptions
+    numpy.seterr(all='raise')
```

Comment:
This is just a concern that may not be relevant.  I saw this line and decided to test some of the behavior myself just out of curiosity.  I was surprised to find out that numpy does *not* raise an exception when you supply a python value to one of its value constructors that would cause an overflow:
```
In [1]: import numpy as np

In [2]: np.seterr(all='raise')
Out[2]: {'divide': 'warn', 'over': 'warn', 'under': 'ignore', 'invalid': 'warn'}

In [3]: np.int8(128)
Out[3]: -128
```
Of course, it does raise an exception when an arithmetic operation causes an overflow:
```
In [4]: np.int8(64) * np.int8(2)
---------------------------------------------------------------------------
FloatingPointError                        Traceback (most recent call last)
<ipython-input-4-3712cff83f6c> in <module>
----> 1 np.int8(64) * np.int8(2)

FloatingPointError: overflow encountered in byte_scalars
```
Just thought I'd point out that behavior in case it was overlooked or in case you were acting on any incorrect assumptions.